src/views/App/App.scss

```diff
@@ -682,4 +682,10 @@ h1 {
 .react-tagsinput{
   border-radius: 3px;
   margin-bottom: 15px;
+}
+.ah-follow-button{
+  font-size: 0.8rem;
+  padding-top: 0.1em !important;
+  padding-bottom: 0.1em !important;
+
 }
```

Comment:
What is missing here?

Reply:
@kimotho-njoki Done

Reply:
@kimotho-njoki Nothing, we have removed the empty line

Reply:
You have removed the empty line? Why?